ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+How big is it going to be?
+What materials are you going to use?
+What layout and design is it going to have?
+
+You also need to think about what you intend for your documentation.
+Is it going to be comprehensive and detailed, or simple and sparse?
+Is it going to use simple language, designed for people who have never done this before, or more technical language, for people who know what they're doing and just need the basics?
+Is it going to start from the very beginning, or are you going to assume that your readers already have some knowledge?
+
+## Face reality
+
+By now, you should have a good idea of what your ideal documentation looks like.
+It will be a shining beacon of docs goodness, hitting all the critical paths for all your readers, and people will talk about it for years to come!
+
+It's a grim reality that project documentation needs to be maintained.
+Have a think about how many writers your project might be able to access.
+Are you likely to have writers on the project over time?
+What skill level are they likely to have?
+Is there any content that already exists elsewhere that you can point to, rather than writing it yourself?
+Is there any way you can automate any part of the documentation, such as using code comments to generate API documentation?
+
+## Choose a direction
+
+Now is when you really get to start making decisions.
+You've thought about your readers, and the kinds of things they might want to do.
+You've thought about the kind of content you already have, and what's missing.
+Then you faced reality and realized that you might not be able to create perfect docs, but you can certainly improve what you already have.
+
+So, which docs are you going to write?
+
+Now you get to decide how to move forward.
+
+## Measure the distance
+
+Don't let your docs project be like your New Year resolutions, forgotten and lonely by the end of the year.
+The best way to make sure you stick to your docs goals, is to work out how much you can do, and set a deadline for each step in the process.
+
+The templates will help you understand how much work is involved in each of the docs you want to write.
+Have a read through the information for each template you want to use, and work out how much time you are going to need to work on it.
+
+Then you can split up docs work like this:
+
+* Planning - 20%
+* Drafting - 50%
+* Reviewing and editing - 20%
+* Production - 10%
+
+So, if you want to write a How To, and you have five months to do it, you can work out how much time to spend on each task like this:
+
+Three months = Around 100 working days, not counting weekends, and allowing for a few holidays.
+You can adjust this however you like, if you're only working on weekends, for example, or if you have other projects to work on as well.
+
+* Planning - 20 days (4 weeks)
+* Drafting - 50 days (10 weeks)
+* Review & edit - 20 days (4 weeks)
+* Production - 10 days (2 weeks)
+
+There's just one more thing to do.
+
+## Prepare to adjust
```

Comment:
hhmm, I think are some valid points here worth expanding.  but not sure if they all work here at the end.  And if this a good closeout.  I like the idea of maybe closing with the process of planning that you have above in __Measure The Distance__